src/service.rs

```diff
@@ -57,6 +57,22 @@ pub struct BlockInfo<Types: NodeType> {
     pub offered_fee: u64,
 }
 
+#[derive(Debug)]
+pub struct BuiderStatesInfo<Types: NodeType> {
+    pub vid_commitments: Vec<VidCommitment>,
```

Comment:
This still does not tell me anything. The name might be ok but I at least need a comment tell me what this is. Of all the VID commitments in the universe what is special about this list we've collected here?

Reply:
https://github.com/EspressoSystems/hotshot-builder-core/pull/120/commits/1fe4bfb9ee3497fa449375ba08adebf788a5a04a

Reply:
this mostly holds when we have malicious leader I guess